fluent/migrate/repo_client.py

```diff
@@ -28,13 +28,13 @@ def __init__(self, root: str):
         self.root = root
         if isdir(join(root, ".hg")):
             self.hgclient = hglib.open(root, "utf-8")
-        elif isdir(join(root, ".git")):
+        else:
             self.hgclient = None
             stdout = git(self.root, "rev-parse", "--is-inside-work-tree")
```

Comment:
This throws if run outside a git repo.
```suggestion
            try:
                stdout = git(self.root, "rev-parse", "--is-inside-work-tree")
            except Exception:
                stdout = ""
```